sbnanaobj/StandardRecord/SRPFP.cxx

```diff
@@ -14,6 +14,7 @@ SRPFP::SRPFP()
     , parent_is_primary(false)
     , trackScore(-5.f)
     , slcID(-5)
+    , t0(std::numeric_limits<float>::signaling_NaN())
```

Comment:
Probably this could use kSignalingNaN from SRConstants (https://github.com/SBNSoftware/sbnanaobj/blob/f37109a218cc8e2920b1fcbc5941808194261848/sbnanaobj/StandardRecord/SRConstants.h#L8).

Other than that and maybe wanting to make an Issue for looking at nesting issues, I think this PR itself seems fine, pending any CI/Validation that needs to take place.

Reply:
fixed.

Reply:
I did not make such an issue. I guess it was a potential suggestion more than an "I did this" :)